examples/virtual-dom-wip/platform/Html/Internal/Client.roc

```diff
@@ -778,7 +778,7 @@ expect
 
 # initClientAppHelp
 expect
-    State : { answer : Nat }
+    State : { answer : U64 }
```

Comment:
this is client side where Nat is U32

Reply:
although to be fair it doesn't matter in this case!

Reply:
Addressed in [735d7cb](https://github.com/roc-lang/roc/pull/4974/commits/735d7cb4df18d848af2cd37aab43d9bcfc90aa12)